build_test.go

```diff
@@ -184,6 +184,69 @@ func testBuild(t *testing.T, when spec.G, it spec.S) {
 				h.AssertNil(t, err)
 				h.AssertEq(t, fakeLifecycle.Opts.AppDir, absPath)
 			})
+
+			it("resolves relative symbolic links", func() {
+				appDir := filepath.Join("testdata", "some-app")
+				absoluteAppDir, err := filepath.Abs(appDir)
+				h.AssertNil(t, err)
+
+				relLinkRef := "./some-app"
+				relLink := filepath.Join("testdata", "some-app.link")
+
+				_ = os.Remove(relLink)
+				h.AssertNil(t, os.Symlink(relLinkRef, relLink))
```

Comment:
If we are going to make temporary file for testing we should make them in a temp dir instead of in the repo. We generally only use the testdata dir if we are going to commit the files.

Reply:
_Full disclosure: I did feel a tad dirty doing it this way._  

I initially attempted to follow the prior practice of using fixtures but when it came time to submit the PR I remembered that git doesn't play nice with symlinks. I obviously should have choose a different path. Thanks for the redirection. Hope it's better now.